index.js

```diff
@@ -3,6 +3,124 @@ const nav = document.querySelector('.nav-wrapper');
 const closeBtn = document.querySelector('.close-menu');
 const ancreLinks = nav.getElementsByTagName('a');
 
+const theModal = document.querySelector('#myModal');
+const closeModal = document.querySelector('.closeModal');
```

Comment:
- [ ] nice work. please kindly reduce the height of the mobile popup so it the buttons and close icon can be seen. this will align it well with the figma design.
- [ ] you are almost there. kindly add the margin bottom under the see live and see source so it follows the figma design
- [ ] please align the description text with the image. nice work

yours
![Screenshot (686)_LI](https://user-images.githubusercontent.com/65298048/208942635-290fb803-0464-4d8f-9d6e-b5a2fd68ac08.jpg)
figma
![Screenshot (684)](https://user-images.githubusercontent.com/65298048/208942865-008dfee8-36df-4432-a784-2f7eab60ac77.png)